pkg/util/nativeutils/rcudp/config.go

```diff
@@ -26,7 +26,7 @@ const (
 	// Encoder configs.
 
 	// BlockSize - max length of an encoding symbol that can fit into a single wire packet.
-	BlockSize = maxUDPLength - 8 - 2 - 2 - 4 - 4
+	BlockSize = maxUDPLength - 8 - 2 - 2 - 4 - 4 - 8
```

Comment:
It seems like this is knocking off all the packet fields from the max UDP frame size. In this case, shouldn't you use `- 1` for the `bcastHeightSize` field instead?

Reply:
Seconded. I doubt that we'd need more than one byte for the block size height

Reply:
No, it cannot be 1 byte.  As per RFC 5053 `- The symbol size, T, in bytes, which MUST be a multiple of Al` .